addons/l10n_latam_invoice_document/models/account_journal.py

```diff
@@ -31,3 +31,10 @@ def check_use_document(self):
             if rec.env['account.move'].search([('journal_id', '=', rec.id), ('state', '!=', 'draft')]):
                 raise ValidationError(_(
                     'You can not modify the field "Use Documents?" if there are validated invoices in this journal!'))
+
+    @api.onchange('type', 'l10n_latam_use_documents')
+    def _onchange_type(self):
+        res = super()._onchange_type()
```

Comment:
I am not sure there is a super method that exists.  @jjscarafia

Reply:
Do not forget to re-target to 13.0 also.  (new pr needed)

Reply:
yes, it's [here](https://github.com/odoo/odoo/blob/13.0/addons/account/models/account.py#L783)

@zaoral will make the new PR's against v13

Thanks!

Reply:
Apparently I was wrong about the new pr...  (you can just retarget)